smn_charts.py

```diff
@@ -0,0 +1,62 @@
+import argparse
+import charts.data_utils as util
+from charts.draw import write_svg, write_pdf
+
+
+def get_args():
+    parser = argparse.ArgumentParser(description='Convenient pdf and svg charts for smn_caller output.')
+
+    parser.add_argument('-p', '--pop-data',
+                        dest="pop_data",
+                        help='smn_caller.py output: Population tsv file',
+                        required=True)
```

Comment:
Can we include the SMN_1kGP_results.tsv with the code, e.g. put it in charts/data/, and make this parameter optional? The distribution should be stable and no need to require user input, unless the user wants to examine their own distribution.